src/Pact/Typechecker.hs

```diff
@@ -846,7 +868,15 @@ toAST :: Term (Either Ref (AST Node)) -> TC (AST Node)
 toAST TNative {..} = die _tInfo "Native in value position"
 toAST TDef {..} = die _tInfo "Def in value position"
 toAST TSchema {..} = die _tInfo "User type in value position"
-toAST TModRef {..} = die _tInfo "TODO: modrefs should be treated as values"
+toAST t@TModRef{..} = do
+  tcid <- freshId _tInfo $ asString $ showPretty t
```

Comment:
```suggestion
  tcid <- freshId _tInfo $ renderCompactText t
```

Reply:
Done (not taking the suggestion, because it doesn't compile)